mono/mini/exceptions-amd64.c

```diff
@@ -337,7 +337,7 @@ mono_arch_get_restore_context_full (guint32 *code_size, MonoJumpInfo **ji, gbool
 	amd64_mov_reg_reg (code, AMD64_R11, AMD64_ARG_REG1, 8);
 	fregs_offset = G_STRUCT_OFFSET (MonoContext, fregs);
 	for (i = 0; i < AMD64_XMM_NREG; ++i) {
-		amd64_movsd_reg_membase (code, i, AMD64_R11, fregs_offset + (i * sizeof (MonoContextSimdReg)));
+		amd64_sse_movups_reg_membase (code, i, AMD64_R11, fregs_offset + (i * sizeof (MonoContextSimdReg)));
```

Comment:
The movsd variant moves one double precision value from an offset in memory into a register.
http://x86.renejeschke.de/html/file_module_x86_id_204.html

The movups variants does the same for 4 packed single precision values, or can be used just to move 128-bits into an SSE register: 
http://x86.renejeschke.de/html/file_module_x86_id_208.html